integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeBlockchainApi.kt

```diff
@@ -67,11 +67,11 @@ open class CrowdNodeBlockchainApi @Inject constructor(
         it.outputs.filter { output ->
             ScriptPattern.isP2PKH(output.scriptPubKey) &&
                 Address.fromPubKeyHash(
-                walletData.wallet!!.params,
+                walletData.networkParameters,
                 ScriptPattern.extractHashFromP2PKH(output.scriptPubKey)
             ) == accountAddress
         }.forEach { output ->
-            walletData.wallet!!.lockOutput(output.outPointFor)
+            walletData.lockOutput(output.outPointFor)
```

Comment:
As per the original idea, feature modules shouldn't use the wallet directly. At some point it'll be moved from the WalletApplication to a dedicated holder class